client/navigation/components/container/style.scss

```diff
@@ -1,6 +1,6 @@
 .woocommerce-navigation {
-	h2 {
-		color: inherit; // This should be handled at component level.
```

Comment:
cc @psealock To make sure this doesn't break anything.  I didn't notice any changes after removing this, but wanted to double check.

Reply:
I think this was left over from Gutenberg Nav being in a state of flux. We should try it with latest Gutenberg activated and just using the version on WP 5.6 to double check. Otherwise, good to remove